client/signup/translations/seller-experience.js

```diff
@@ -3,7 +3,7 @@ import { translate } from 'i18n-calypso';
 
 // Set up your store
 translate( 'Set up your store' );
-translate( 'Let’s create a website that suits your needs.' );
+translate( "Let's make sure you have everything you need to sell online." );
```

Comment:
Does it make a difference if we use a straight quote rather than a fancy quote? I don't know if we have a standard or if it comes across the same for translators. (See line 8 for an example.)

Reply:
Haha I'm two minutes late to the fancy quote/straight quote party, Andy commented already 😆

Reply:
They're different characters, and the apostrophe is technically correct (but I'm not sure I've seen it elsewhere in Calypso). @Automattic/editorial Do we prefer apostrophes over single quotes when appropriate, like this case?..

Reply:
Single-quote formatting isn't covered explicitly in the [WordPress.com style guide](https://docs.google.com/document/d/1dOydmo9EqImcyfEg2WksEh6hBYtFKFitpiBGKbImXCY/edit), but I certainly see it pop up in i11n discussions: https://i18n.wordpress.com/2020/11/10/issues-with-apostrophes-in-french-translations. Because typing a curly apostrophe requires three keys (shift-option-]), I'm inclined to stick with using the single quote for ease of input.

Reply:
> I think WordPress auto-converts straight to curly anyway?

Good question – core does, but I'm not sure that happens with Calypso and API responses? Regardless, we can stick to single quotes and drop the literal apostrophe usage. 👍